core/base/src/main/java/alluxio/security/authorization/Mode.java

```diff
@@ -229,6 +230,30 @@ public String toString() {
     return mOwnerBits.toString() + mGroupBits.toString() + mOtherBits.toString();
   }
 
+  /**
+   * Creates {@link Mode} from proto {@link alluxio.grpc.Mode}.
+   *
+   * @param pMode proto mode
+   * @return created mode
+   */
+  public static Mode fromProto(alluxio.grpc.Mode pMode) {
```

Comment:
Should we call it `PMode`, or are we no longer prefixing with `P` to avoid name collisions?

Reply:
We can make it PMode.